src/olympia/addons/models.py

```diff
@@ -2169,6 +2169,14 @@ class ReplacementAddon(ModelBase):
     class Meta:
         db_table = 'replacement_addons'
 
+    @staticmethod
+    def is_external_path(path):
+        scheme = path.split('://')[0].lower()
```

Comment:
Should this use urlparse?

Reply:
`path.split('://')[0].lower()` is what `validators.URLValidator` does internally so I matched it before it was called in `admin.py::ReplacementAddonForm`.  But probably, yes.